javascript/MaterialXView/source/index.js

```diff
@@ -21,7 +21,15 @@ let normalMat = new THREE.Matrix3();
 let viewProjMat = new THREE.Matrix4();
 let worldViewPos = new THREE.Vector3();
 
-const materialFilename = new URLSearchParams(document.location.search).get("file");
+// Get URL options. Fallback to defaults if not specified.
+let materialFilename = new URLSearchParams(document.location.search).get("file");
```

Comment:
This is kind not efficient the way it's implemented. It reloads the entire page with option args which specify the material, and now the geometry file. Refactoring this is out-of-scope for this PR.